### ISSUE TITLE
No longer connecting

### ISSUE THREAD

**Issue by wookienow** [bwssytems/ha-bridge#133]
Hi,
Has anything changed with the logictech connection. I had this working and now I am getting a connection error.
It successfully get this the User Token 
 DEBUG n.w.harmony.protocol.AuthService - auth token request returned: {"GetUserAuthTokenResult":{"AccountId":0,"UserAuthToken":"XXX"}}

but can't login into the device with this error.
Exception in thread "main" java.lang.RuntimeException: Failed communicating with Harmony Hub
        at net.whistlingfish.harmony.HarmonyClient.connect(HarmonyClient.java:182)
        at com.bwssystems.restfulharmony.HarmonyRestServer.execute(HarmonyRestServer.java:66)
        at com.bwssystems.restfulharmony.HarmonyRestServer.main(HarmonyRestServer.java:49)
Caused by: org.jivesoftware.smack.sasl.SASLErrorException: SASLError using PLAIN: not-authorized
        at org.jivesoftware.smack.SASLAuthentication.authenticate(SASLAuthentication.java:348)
        at org.jivesoftware.smack.tcp.XMPPTCPConnection.login(XMPPTCPConnection.java:244)
        at net.whistlingfish.harmony.HarmonyClient.connect(HarmonyClient.java:119)

I can successfully ping the device and connect to it from the logitech app with the same user name and password. And can ping from the server running the ha-bridge. I have tried both the ha-bridge and the Harmony rest server.

Any ideas on how to further debug.

**Comment by audiofreak9** [comment]
Stupid question but have you checked to make sure the Harmony IP in the Bridge Control tab is correct?

**Comment by wookienow** [comment]
Not stupid at all. But yes I have.
I have also tried using other restful-harmony java app passing IP via the command line with the same result.
It was just working and then it wasn't. 
I can still control via the harmony app though.

**Comment by bwssytems** [comment]
Interesting, when did this so working and also the version of the Harmony
hub software.

Thanks

**Comment by wookienow** [comment]
4.1.3

**Comment by wookienow** [comment]
Sorry it is 4.10.30. it started happening. A day ago.

**Comment by audiofreak9** [comment]
Just so you can compare, I am running ha-bridge Ver 2.0.7 on a Raspberry Pi, with a Harmony Hub running version 4.10.30 without issue.

**Comment by wookienow** [comment]
Thanks. Exactly what I am. What version of Java? Who knows what is causing it then.

**Comment by audiofreak9** [comment]
Java build 1.8.0_65-b17, best of luck!

**Comment by audiofreak9** [comment]
Also my RPi is hard wired, not wireless with a static IP- just trying to think of all possible differences.

**Comment by bwssytems** [comment]
Is this working now or still an issue?

**Comment by wookienow** [comment]
Hi,
Thanks for all your help. When all of your versions and everything matched mine thought it must have been something else.
I rebuilt my entire Pi. This didn't work.
I ended up factory resetting my Harmony Hub and setting it up with a completely separate account. This didn't work.
I then moved the device to a new fixed IP on the network and it started working again. I am not sure why it stopped but it is up and running again.
Thanks for providing all the details of what you had. It gave me hope that I could get it working again and that Harmony hadn't broken something.
